tasks/setup-sa- client.yaml

```diff
@@ -0,0 +1,62 @@
+kind: Task
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: jumpstarter-setup-sa-client
+spec:
+  description: |
+    Create Jumpstarter client config from the user system account in Kubernetes,
+    this requires that Jumpstarter is configured to trust authentication from K8s
+  params:
+    - name: endpoint
+      description: "The Jumpstarter grpc endpoint"
+      type: string
+    - name: namespace
+      description: "The Jumpstarter client namespace"
+      type: string
+    - name: name
+      description: "The Jumpstarter client name"
+      type: string
+    - name: insecure-tls
+      description: "Use insecure tls for grpc"
+      type: string
+      default: "false"
+    - name: tls-ca
+      description: "CA Cert for the TLS endpoint"
+      type: string
+      default: ""
+
+  results:
+    - name: config
+      description: "The content of the generated Jumpstarter client config file"
+
+  steps:
+    - name: setup-client
+      image: quay.io/fedora/fedora-minimal:43
+      script: |
+        #!/usr/bin/env bash
+
+        CONFIG_DIR=$(workspaces.config-dir.path)
+
+        # generate jumpstarter client config
+        cat > ${CONFIG_DIR}/default.yaml <<EOF
+        apiVersion: jumpstarter.dev/v1alpha1
+        kind: ClientConfig
+        metadata:
+          namespace: $(params.namespace)
+          name: $(params.name)
+        endpoint: $(params.endpoint)
+        tls:
+          insecure: $(params.insecure-tls)
+          ca: \"$(params.tls-ca)\"
+        token: __POD_TOKEN__
```

Comment:
where is this replaced?

Reply:
https://github.com/rh-sdv-cloud-incubator/aib-devspaces/pull/17 in the tasks here

Reply:
Yep, later in tasks, so we always have a fresh token.